### DISCUSSION TITLE
How to install verdaccio without npm/docker?

### DISCUSSION THREAD

**Discussion by Noriller** [verdaccio/verdaccio#3081] [Q&A]
I have this offline machine I want to install verdaccio to.
I can and will probably have a mirror verdaccio in my online machine and then sync with the offline one.

My problem right now is on how to install verdaccio there.  I've tried to make a blank project, install verdaccio there and install from that node modules folder, also from the storage in verdaccio, but, probably because of the dependencies it won't install them.

So, how can I download and then install it totally offline?

**Comment by hardillb** [comment]
If you still want to use docker, but just on the offline machine you can pull the container to a online machine and then export to a tar file that can be copied to the offline machine and then imported.

e.g. `docker save -o verdaccio.tar verdaccio/verdaccio` then use the `docker import` command on the offline machine

With pure npm then using something like https://www.npmjs.com/package/npm-pack-all to bundle all the dependencies into a tgz that can then be copied over and the installed with npm.
